front/lib/api/extract.ts

```diff
@@ -124,21 +127,70 @@ export async function updateEventSchema(
     description: schema.description,
     status: schema.status,
     properties: schema.properties,
+    workspaceId: schema.workspaceId,
   };
 }
 
-export async function getExtractedEvents(
-  auth: Authenticator,
-  schemaId: number
-): Promise<ExtractedEventType[]> {
+export async function getExtractedEvent({
+  auth,
+  marker,
+  eventId,
+}: {
+  auth: Authenticator;
+  marker: string;
+  eventId: string;
+}) {
```

Comment:
We should return EventType here not the model?

Reply:
Indeed 😱